elide-graphql/src/main/java/com/yahoo/elide/graphql/GraphQLEndpoint.java

```diff
@@ -135,8 +135,10 @@ public Response post(
 
             return Response.ok(mapper.writeValueAsString(result)).build();
         } catch (JsonProcessingException e) {
+            log.debug("Invalid json body provided to GraphQL", e);
             return buildErrorResponse(new InvalidEntityBodyException(graphQLDocument), isVerbose);
         } catch (IOException e) {
+            log.error("Uncaught IO Exception by Elide in GraphQL", e);
```

Comment:
Shouldn't we log the ForbiddenAccessExceptions here (or are these already logged internally)?

Reply:
Good question. We do log forbidden access elsewhere, but we can throw in a debug here too just for completeness. Adding now